web-wallet/src/routes/(app)/dashboard/+page.svelte

```diff
@@ -77,6 +78,21 @@
     tokens={balance.value}
   />
 
+  {#if hasNoEnabledContracts}
```

Comment:
This does not account for the case where the ENV variables are not even defined.
If `VITE_CONTRACT_STAKE_DISABLED` or `VITE_CONTRACT_TRANSFER_DISABLED` are not present the app considers that both or just one of them is enabled.

Reply:
You're right. I will think on how to handle this (in the best way).

Reply:
I would add extra validation here https://github.com/dusk-network/rusk/blob/93c3ea71ade64d8ae6442734429a76aa3099397e/web-wallet/src/lib/contracts/contract-descriptors.js#L10 as this is the main setter of the enabled/disabled contracts.

Reply:
I do agree, it contains the logic in one place. Have a look [now](https://github.com/dusk-network/rusk/pull/1427/files#diff-abdab69d356c20c44fdc6355770a9b7d43d87c6e2f733899c867de12aedba8faR10-R12).

Thanks again for spotting this! 👏